### ISSUE TITLE
VSCode does not register changes in files under directories with non-conventional characters

### ISSUE THREAD

**Issue by albireox** [microsoft/vscode#83484]
- VSCode Version:   1.40.0-insider
- OS Version: 10.15

Steps to Reproduce:

1. Access a file in a directory with non-conventional characters (e.g., `{{cookiecutter.repo_name}}`)
2. Modify a file.
3. The editor doesn't seem to register the fact that the file has changed. Saving the file does not save it to disk. Somehow, doing Save All does save it to file.

Does this issue occur when all extensions are disabled?: Yes

**Comment by Tyriar** [comment]
Do you have a single or multiple folders opened?

**Comment by albireox** [comment]
Interestingly, the problem seems to have just disappeared. I reopened the project that was giving me problems and the issue was still there. After closing all the tabs and refreshing the explorer everything seems fine now.

I'm fairly sure I did all that a few times before opening the issue. I wonder if the overnight version of Insiders somehow fixed this. I started noticing the problem after cherry-picking some files with conflict between two branches in the project; it may also be possible that had something to do.

Since I cannot reproduce anymore I think it's ok to close this and I'll reopen if it happens again.
